ara/cli/result.py

```diff
@@ -34,13 +34,45 @@ class ResultList(Lister):
     """Returns a list of results"""
     log = logging.getLogger(__name__)
 
-    def take_action(self, parsed_args):
+    def get_parser(self, prog_name):
+        parser = super(ResultList, self).get_parser(prog_name)
+
+        g = parser.add_mutually_exclusive_group(required=True)
+        g.add_argument(
+            '--playbook', '-b',
+            metavar='<playbook-id>',
+            help='Playbook from which to list results',)
+        g.add_argument(
+            '--play', '-p',
+            metavar='<play-id>',
+            help='Play from which to list results',)
+        g.add_argument(
+            '--task', '-t',
+            metavar='<task-id>',
+            help='Task from which to list results',)
+        g.add_argument(
+            '--all', '-a',
```

Comment:
I don't think that listing "all results in the database" makes any sense as the default behavior.  After a few runs and several playbooks, you will just get a bunch of information you probably don't care about.